### PR TITLE
Tweak: Wrap Pages Panel in experiment [ED-11346]

### PR DESCRIPTION
**Pull request by lenalElementor**

### CODE REVIEW

File: modules/site-navigation/module.php [hunk 1]
```diff
@@ -21,6 +23,15 @@ class Module extends Module_Base {
 	public function __construct() {
 		Plugin::$instance->data_manager_v2->register_controller( new Controller() );
 
+		if ( Plugin::$instance->experiments->is_feature_active( self::PAGES_PANEL_EXPERIMENT_NAME ) ) {
+			add_filter( 'elementor/editor-v2/packages/client-env', function ( $env ) {
+				$env['@elementor/editor-site-navigation'] = [
+					'is_panel_active' => true,
```
Comment (ManorHazazE) [inline modules/site-navigation/module.php:29]: Maybe?
If you agree you need to make sure to change it in the packages too.

```suggestion
					'is_pages_panel_active' => true,
```

File: modules/site-navigation/module.php
Comment (ManorHazazE) [inline modules/site-navigation/module.php:1]: Tests?

**Review by ghost — CHANGES_REQUESTED** [review]

**Review by ghost — CHANGES_REQUESTED** [review]
Tests are still missing?

File: tests/phpunit/elementor/modules/site-navigation/test-module.php [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+		Plugin::$instance->experiments->set_feature_default_state(
+			SiteNavigation::PAGES_PANEL_EXPERIMENT_NAME,
+			$this->original_experiment_default_state
+		);
+	}
+
+
+	public function test_construct_experiment_inactive() {
+		remove_all_filters( 'elementor/editor-v2/packages/client-env' );
```
Comment (ManorHazazE) [inline tests/phpunit/elementor/modules/site-navigation/test-module.php]: You can put it in the setup

**Review by ghost — APPROVED** [review]

**Pull request merged**